plugins/source/k8s/Makefile

```diff
@@ -1,6 +1,7 @@
 # Generate mocks for mock/unit testing 
 .PHONY: generate-mocks
 gen-mocks:
+	go install github.com/golang/mock/mockgen@v1.6.0
```

Comment:
We can do this only in the GitHub Action workflow, but seems useful to do it for everyone